kroxylicious/src/main/java/io/kroxylicious/proxy/internal/package-info.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/**
+ * <b>io.kroxylicious.proxy.internal</b> contains Kroxylicious internal code implementing Proxying functionality using Netty.
+ * <h2>Understanding the Kroxylicious Pipeline</h2>
+ * The Kroxylicious pipeline for a client connection is composed of:
+ * <ul>
+ *     <li>A {@link io.kroxylicious.proxy.internal.KafkaProxyInitializer} that initialises the incoming channel pipeline</li>
+ *     <li>A {@link io.kroxylicious.proxy.internal.KafkaProxyFrontendHandler} that establishes an outgoing channel pipeline to the upstream broker</li>
```

Comment:
Aside - do you think we should name KafkaProxyFrontendHandler -> KafkaProxyUpstreamHandler?

Reply:
No, ignore this.

Reply:
I dunno I'm sort of tempura agree. I'm not sure upstream handler is the right term but this is the only thing in our "vocabulary" that mentions "backend" iirc